content/syllabuses/data-sci.md

```diff
@@ -42,7 +42,8 @@ _db_id: 2
 - {{% contentlink path="specific-skill-success-criteria/classes-and-objects" flavour="python" %}}
 - {{% contentlink path="projects/data-science-specific/oop-for-data-science" flavour="python" %}}
 - {{% contentlink path="topics/solo-learn/python/10-pythonicness-and-packaging-project" %}}
-- {{% contentlink path="projects/github-api-consume" flavour="python" %}}
+- {{% contentlink path="projects/github-api-consume-part-1" flavour="python" %}}
+- {{% contentlink path="projects/github-api-consume-part-2" flavour="python" %}}
```

Comment:
Is part 2 necessary for the data science guys? If yes, that's perfect, ignore this comment.

Reply:
@Andy-Nkumane , will it be necessary for data science?

Reply:
@Sbonelo01 i don't think it should, part 1 should be enough as their main focus should be on modelling projects